.buildkite/pipeline.yml

```diff
@@ -76,6 +76,24 @@ steps:
       CMAKE_FLAGS: "-DCMAKE_TOOLCHAIN_FILE=cmake/darwin-x86_64.cmake"
       RUN_TESTS: "false"
 
+
+  - label: "Build :cpp: for Windows_x86_64-RelWithDebInfo"
+    timeout_in_minutes: 180
+    agents:
+      provider: gcp
+      image: family/ci-windows-2016
+    commands:
+      - 'echo "checkout succeeded"'
+      - Get-ChildItem
+      - "dev-tools\\download_windows_deps.ps1"
+      - 'Install-Binary -Url "https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&channel=Release&version=VS2022&source=VSLandingPage&cid=2030&passive=false" -Name "VisualStudioSetup.exe" -ArgumentList ("/quiet")'
```

Comment:
We shouldn't be installing our dependencies on every build. Doesn't `family/ci-windows-2016` already have this installed? If not, the solution is to use the right base image. If we could use the base image we use in Jenkins that would be perfect.

Reply:
Try `family/elastic-elasticsearch-ci-windows-2016-ml-pinned`

Reply:
Sadly trying `family/elastic-elasticsearch-ci-windows-2016-ml-pinned` results in:

```
googleapi: Error 404: The resource 'projects/elastic-images-qa/global/images/family/elastic-elasticsearch-ci-windows-2016-ml-pinned' was not found, notFound
```

The VM images must now be defined here in the new `ci-agent-images` repo by way of extending one of the many existing base images there.